### DISCUSSION TITLE
Privacy Violations ?

### DISCUSSION THREAD

**Discussion by helpfulgent** [organicmaps/organicmaps#5035] [Q&A]
If OrganicMaps has such high privacy - 
why does it require phone Location enabled during install ?

There is no need ever to have Location installed - unless or unless 
the device operator Chooses to do so.

Some of us want maps withOUT having to give up our location.
In some cases, like a war zone, a life could be risked by giving out Location.

**Comment by biodranik** [comment]
You can choose if you want to give permission to access the device's location or not. And then use Organic Maps without seeing your current location on the map.

But what is your use case? How is privacy related to this scenario? Organic Maps doesn't share your location with anyone. It is used only to display your position on the map.
